lint/testdata/importPackageName/positive_tests.go

```diff
@@ -0,0 +1,18 @@
+package checker_test
+
+import (
+	/// unnecessary rename of import package
+	fmt "fmt"
+
+	/// unnecessary rename of import package
+	lint "github.com/go-critic/go-critic/lint"
+
+	/// unnecessary rename of import package
+	dummypkg "github.com/go-critic/go-critic/lint/internal/dummy"
```

Comment:
Why `dummy` => `dummypkg` rename is unnecessary?

Reply:
'Cause it's name is `dummypkg` already :)

Reply:
@cristaloleg is right. From the [spec](https://golang.org/ref/spec#Import_declarations):

> If the PackageName is omitted, it defaults to the identifier specified in the package clause of the imported package.